events.js

```diff
@@ -12,19 +12,19 @@ function Event(id, title, desc, date) {
 let arr = [];
 
 for(let i = 0; i < 10; i++) {
-  arr.push(new Event(i, `title ` + i, 'desc ' + i, 'date ' + i));
+  arr.push(new Event(i, `title ` + i, 'desc ' + i, new Date().toString()));
 }
 
 router.get('/', (request, response, next) => {
   response.send(arr);
-})
+});
 
 router.get('/:id', (request, response, next) => {
   const id = parseInt(request.params.id);
   response.send(arr.filter(x => {
     return x.id === id;
   }));
-})
+});
 
 router.post('/', (request, response, next) => {
   newEvent = new Event(request.body.id, request.body.title, request.body.description, request.body.date);
```

Comment:
Mongo/Mongoose will take care of auto generating the IDs once you start using a real database, but it's a good practice to make sure the backend handles the generation of the IDs so the client doesn't have to worry about it. How will the client know if a given ID is unique or not?